Modules/_testcapimodule.c

```diff
@@ -5356,6 +5356,23 @@ test_pyobject_vectorcall(PyObject *self, PyObject *args)
 }
 
 
+static PyObject *
+override_vectorcall(
+    PyObject *callable, PyObject *const *args, size_t nargsf, PyObject *kwnames) {
+    Py_RETURN_NONE;
```

Comment:
Could you modify the test to return the string "overridden" instead of just returning None?